### ISSUE TITLE
IBM accessibility and KC rendered swagger yaml/json files

### ISSUE THREAD

**Issue by to-kra** [swagger-api/swagger-ui#3235]
To be accessible with IBM, we follow the accessibility guidelines and scan our web documents using the Dynamic Assessment Plugin (DAP) for FireFox; which is an automated test tool that is required to test accessibility complaisance of all Web content rendered in a standard browser. To be compliant, all violations (V), if any are reported, must be fixed to comply with the Web accessibility checklist. Unfixed violations result in a failure (non-compliance) with the associated checkpoints.
 
Using the DAP tool, the swagger docs include a few violations (see image below)
![image](https://user-images.githubusercontent.com/2188084/27088576-584f2574-5058-11e7-9889-a343959c72ce.png)

**Comment by shockey** [comment]
`v2.2.10` is legacy - our current releases are 3.x 😄 

would be happy to look at an accessibility report for our current version, though.

**Comment by to-kra** [comment]
@shockey current version 3.x is not supporting translations yet, which is very important for us.

**Comment by to-kra** [comment]
@shockey I just updated swagger-ui to latest 3.x and content team ran accessibility scan, and they found even more foundings. is there are possibility i could send you guys these results/symptoms somehow privately, i dont want to put them here. what you think ? thanks

**Comment by shockey** [comment]
@to-kra, sure, my email is in my profile - I can distribute it to our core team. Thanks for taking the time to put that together!

**Comment by to-kra** [comment]
@shockey i sent you email with accessibility scan results.

**Comment by to-kra** [comment]
@shockey any update on this one ?

**Comment by shockey** [comment]
@to-kra no updates unfortunately - it will likely be a while before we can carve out some time dedicated to accessibility. I did get your email though, the report is very helpful!

If you or your team has some extra time, a pull request that improves the situation would be very welcome 😄

**Comment by mgifford** [comment]
@shockey it has been over 5 years. Would love to see more movement on this and other accessibility issues.   
  
There are lots of updates from [https://www.ibm.com/able/](https://www.ibm.com/able/)  
  
But also tools like [https://accessibilityinsights.io/](https://accessibilityinsights.io/) & WAVE that can track down issues.
